input/pages/codings.xml

```diff
@@ -194,8 +194,8 @@
+    <td>HLA Nomenclature<br/>(European Bioinformatics Institute)</td>
+    <td>Human leukocyte antigen (HLA) complex contains more than 220 genes that encode for the proteins of the immune system. HLA alleles are most commonly used for histocompatibility testing for stem cell and solid organ transplantation. The WHO Nomenclature Committee for Factors of the HLA System is responsible for a common nomenclature of HLA alleles, allele sequences, and quality control, to communicate histocompatibility typing information to match donors and recipients. An HLA allele is defined as any set of variations found on a sequence of DNA comprising a HLA gene. So, if there are five variations found in this one gene sequence, this set is defined as one allele (vs. the definition of an allele being the variation found between the test specimen and the reference along a contiguous stretch of DNA). In the case of HLA, the contiguous stretch of DNA represents the entire gene, and the variations do not need to be contiguous within the gene sequence. Each HLA allele name has a unique name consisting of the gene name followed by up to four fields, each containing at least two digits, separated by colons. There are also optional suffixes added to indicate expression status. For the full specification, please go to this website: http://hla.alleles.org/nomenclature/naming.html.HLA nomenclature can also be used to represent sets of alleles that share sequence identity in the Antigen Recognition Site (ARS). G-groups are alleles that have identical DNA sequences in the ARS, while P-groups are alleles that have identical protein sequences in the ARS. These are described, respectively, inhttp://hla.alleles.org/alleles/g_groups.html and http://hla.alleles.org/alleles/p_groups.html .</td>
```

Comment:
Looks like this might be missing some spaces:

`naming.html.HLA nomenclature can also`

`inhttp://hla.alleles.org/alleles/g_groups.html`

Extra space before the last period?

Reply:
added